src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/IPackageMetadata.cs

```diff
@@ -48,5 +49,7 @@ public interface IPackageMetadata
         /// Returns sets of Content Files specified in the manifest.
         /// </summary>
         ICollection<ManifestContentFiles> ContentFiles { get; }
+
+        IReadOnlyList<PackageType> PackageTypes { get; }
```

Comment:
Should this be IEnumerable<> like mostly everything else in here? Not sure why only one thing is ICollection, but now there's three different collection interfaces used here.